tpch_2_13_0/distributed_queries/12.sql

```diff
@@ -28,4 +28,5 @@ WHERE
 GROUP BY
     l_shipmode
 ORDER BY
-    l_shipmode;
+    l_shipmode
+LIMIT 4;
```

Comment:
why do we change the queries? These are predefined tpch queries, so it might be nicer not to change

Reply:
Even when we do manual tpch tests, sometimes the output is long and it shows `more` and it waits for user input to continue. So basically if you run a tpch test manually, you cannot finish the test without pressing enter a few times because of this. I have realized this when automating the tests and therefore limited the results a bit to prevent this.

Reply:
https://github.com/citusdata/test-automation/issues/120

Reply:
Then, shall we at least have comments like:
```SQL
....
ORDER BY
    l_shipmode
-- the following LIMIT is not part of the benchmark suite
-- but required to run the tests automatically
-- see https://github.com/citusdata/test-automation/issues/120
LIMIT 5;
```

My goal is to prevent any future users to rely on these queries as being the exact tpch queries.